docs/business-apps/get-started/set-up-sharepoint-site-lists-libraries.md

```diff
@@ -1,20 +1,25 @@
 ---
 title: Set up your SharePoint site with lists and libraries
 description: Set up your SharePoint site with lists and libraries
-ms.date: 12/12/2018
+ms.date: 3/11/2020
 ms.prod: sharepoint
 localization_priority: Priority
 ---
 
 # Set up your SharePoint site with lists and libraries
-
 To successfully complete the *Getting Started* tutorials, set up your SharePoint site with the following items.
 
-## Projects list
+## Microsoft 365 tenant
+To use Power Automate and build flows, you need an Office 365 tenant.
+
+If you already have an Office 365 tenant, see section **Create Projects list** below.
+
+If you don't have one, you can get an Office 365 developer subscription when you join the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program). See the [Office 365 Developer Program documentation](https://docs.microsoft.com/office/developer-program/office-365-developer-program) for step-by-step instructions about how to join the Office 365 Developer Program and sign up and configure your subscription.
+
+## Create Projects list
 Create a new SharePoint list called **Projects**. To create a new SharePoint list, follow these steps.
 
 ### Create a new list
-
 1. In your Microsoft 365 tenant, browse to a modern SharePoint site.
 
 > **Note:** You can use either a modern Teams site or a Communications site.
```

Comment:
Replace **Note:** with [!NOTE]

Reply:
@jackwi-alt trying the markup [!NOTE] is not resulting in anything other than just the text being rendered. Can you let me know what am I missing?

![image](https://user-images.githubusercontent.com/7882052/78713905-d37d7780-78cf-11ea-8ee3-071ae330f3cd.png)